python_modules/libraries/dagster-looker/dagster_looker/api/resource.py

```diff
@@ -28,6 +35,9 @@
 logger = get_dagster_logger("dagster_looker")
 
 
+LOOKER_SOURCE_METADATA_KEY_PREFIX = "dagster-looker/source"
```

Comment:
We're calling it "reconstruction metadata" now so should probably change this for consistency

Reply:
👍 will change